### PR TITLE
Fixed kfctl init commands and Kubeflow version for v0.6.2

### PR DESCRIPTION
**Pull request by sarahmaddox**
Fixes https://github.com/kubeflow/website/issues/1115
Use variables (shortcodes) for config URIs (see the above issue for reasoning).

Fixes https://github.com/kubeflow/website/issues/1116
Fix the kfctl init commands across various docs.

Fixes https://github.com/kubeflow/website/issues/1117
Update the Kubeflow version in the docs to v0.6.2.

Fixes https://github.com/kubeflow/website/issues/1118
Remove the ksonnet-min-version shortcode as it's no longer used.

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1114)

### CODE REVIEW

File: layouts/shortcodes/config-uri-k8s-istio.html [hunk 1]
```diff
@@ -0,0 +1 @@
+https://raw.githubusercontent.com/kubeflow/kubeflow/v0.6-branch/bootstrap/config/kfctl_k8s_istio.0.6.2.yaml
```
Comment (sarahmaddox) [inline layouts/shortcodes/config-uri-k8s-istio.html]: @jlewi Is this the right file for the `kfctl-k8s-istio` config? You can see the result in the preview here:
https://deploy-preview-1114--competent-brattain-de2d6d.netlify.com/docs/started/k8s/kfctl-k8s-istio/

**Comment by sarahmaddox** [comment]
Previews:
* https://deploy-preview-1114--competent-brattain-de2d6d.netlify.com/docs/started/k8s/kfctl-existing-arrikto/
* https://deploy-preview-1114--competent-brattain-de2d6d.netlify.com/docs/started/k8s/kfctl-k8s-istio/
* https://deploy-preview-1114--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-cli/
* https://deploy-preview-1114--competent-brattain-de2d6d.netlify.com/docs/other-guides/kustomize/
* https://deploy-preview-1114--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/monitor-iap-setup/
* https://deploy-preview-1114--competent-brattain-de2d6d.netlify.com/docs/gke/private-clusters/

**Comment by sarahmaddox** [comment]
Note: I haven't updated the Microsoft Azure or AWS instructions.

**Comment by yanniszark** [comment]
Thanks @sarahmaddox, everything looks great!
/lgtm

**Comment by abhi-g** [comment]
Thanks @sarahmaddox for the much needed cleanup.
/lgtm
/approve

**Pull request merged**